### PR TITLE
[FEATURE] Changed dependencies for sulu 1.3

### PR DESCRIPTION
**Pull request by turbo-ele**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | no
| Related issues/PRs | https://github.com/sulu/sulu/pull/2890
| License | MIT
| Documentation PR | none

#### What's in this PR?

Adapts product-bundle to work with sulu version 1.3.

Also multiple cleanups have been performed.

### CODE REVIEW

File: Resources/config/services.xml [hunk 1]
```diff
@@ -210,10 +209,17 @@
...
         <!--PROVIDER-->
-        <service id="sulu.core.localization_manager.core_provider" class="%sulu.core.localization_manager.core_provider.class%">
+        <!--<service id="sulu.core.localization_manager.core_provider" class="%sulu.core.localization_manager.core_provider.class%">-->
+            <!--<argument>%sulu_product.locales%</argument>-->
+
+            <!--<tag name="sulu.localization_provider" alias="HUHU"/>-->
+        <!--</service>-->
+        <service id="sulu_product.localization_provider" class="Sulu\Component\Localization\Provider\LocalizationProvider">
             <argument>%sulu_product.locales%</argument>
+
```
Comment (trickreich) [inline Resources/config/services.xml]: rm empty line

File: Resources/config/services.xml [hunk 2]
```diff
@@ -210,10 +209,17 @@
             <tag name="sulu.context" context="admin"/>
         </service>
         <!--PROVIDER-->
-        <service id="sulu.core.localization_manager.core_provider" class="%sulu.core.localization_manager.core_provider.class%">
+        <!--<service id="sulu.core.localization_manager.core_provider" class="%sulu.core.localization_manager.core_provider.class%">-->
```
Comment (trickreich) [inline Resources/config/services.xml]: ???

File: Resources/public/js/components/products/components/detail-form/main.js [hunk 3]
```diff
@@ -118,7 +118,9 @@ define([
         },
 
         render: function() {
-            this.sandbox.dom.html(this.$el, this.renderTemplate('/admin/product/template/product/form'));
+            this.sandbox.dom.html(this.$el, this.renderTemplate('/admin/product/template/product/form', {
```
Comment (trickreich) [inline Resources/public/js/components/products/components/detail-form/main.js]: use constant for template path

File: Tests/Functional/Controller/ValueControllerTest.php [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <?php
+
 /*
- * This file is part of the Sulu CMS.
+ * This file is part of Sulu.
```
Comment (trickreich) [inline Tests/Functional/Controller/ValueControllerTest.php]: When we need the same header, add the it here also.

File: README.md [hunk 5]
```diff
@@ -1,26 +1,57 @@
...
+```
+"sulu/product-bundle": "~0.12"
+```
+
+Add the following to your application kernel:
+
+```
+// Product bundle
+new Sulu\Bundle\ProductBundle\SuluProductBundle(),
+new Sulu\Bundle\PricingBundle\SuluPricingBundle(),
```
Comment (trickreich) [inline README.md]: I thought you've removed this dependency?

**Comment by trickreich** [comment]
else ack

File: Resources/config/services.xml (hunk 1 above)
Comment (turbo-ele) [inline Resources/config/services.xml]: this emtpy line actually is for the sake of better readability.
it's to clearly separate tags from arguments.
(i now did that for all tags in services.xml, which comes with the next commit)

File: Tests/Functional/Controller/ValueControllerTest.php (hunk 4 above)
Comment (turbo-ele) [inline Tests/Functional/Controller/ValueControllerTest.php]: we've just decided to add style-ci to product-bundle, which will reformat headers automatically in future.

File: README.md (hunk 5 above)
Comment (turbo-ele) [inline README.md]: in another PR ;)

Comment (turbo-ele) [inline README.md]: but of course you are right

**Pull request merged**